server/app/assets/javascripts/main.ts

```diff
@@ -416,17 +416,19 @@ function configurePredicateFormOnOperatorChange(event: Event) {
 }
 
 function attachFormDebouncers() {
-  Array.from(document.querySelectorAll('.cf-debounced-form')).forEach((formEl) => {
```

Comment:
While technically unrelated, this was caught as part of running `bin/fmt`, which didn't catch this file before. As far as I can tell, the only changes here are formatting.